src/chart/Bubble.js

```diff
@@ -48,13 +66,22 @@
             .attr("class", "node")
             .attr("opacity", 0)
             .on("click", function (d) {
-                context.click(context.rowToObj(d), context._columns[1]);
+                    console.log("data", context._data);
```

Comment:
@dtsnell4 @GordonSmith 
indent, + should this inherit IXCHart such that you can do .on("click", this.click); ?